src/systemctl/systemctl.c

```diff
@@ -399,7 +399,17 @@ static int output_units_list(const UnitInfo *unit_infos, unsigned c) {
         if (!table)
                 return log_oom();
 
+        r = table_set_display_all(table);
+        if (r < 0)
+                return log_error_errno(r, "Failed to set columns to display: %m");
```

Comment:
Maybe move this call to table_set_display_all() into table_hide_column_from_display()?
```C
if (!t->display_map)
    table_set_display_all();
```

Reply:
Ahh, nice, not sure why didn't think of this.